specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/serverVulnerabilityAssessments.json

```diff
@@ -0,0 +1,338 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Security Center",
+        "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+        "version": "2019-01-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.Security/ServerVulnerabilityAssessments": {
+            "get": {
+                "x-ms-examples": {
+                    "Get a list of server vulnerability assessments on a resource. Though this API returns a list, Currently Microsoft.Security only supports a single default type of server vulnerability assessment": {
+                        "$ref": "./examples/ServerVulnerabilityAssessments/ListByExtendedResourceServerVulnerabilityAssessments_example.json"
+                    }
+                },
+                "description": "Gets a list of server vulnerability assessment onboarding statuses on a given resource.",
+                "tags": [
+                    "ServerVulnerabilityAssessments"
+                ],
+                "operationId": "ServerVulnerabilityAssessment_ListByExtendedResource",
+                "parameters": [
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "name": "resourceNamespace",
+                        "in": "path",
+                        "description": "The Namespace of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceType",
+                        "in": "path",
+                        "description": "The type of the resource.",
+                        "required": true,
+                        "type": "string"
+                    },
```

Comment:
For extension resources, users can try to add the extension on a resource that makes no sense. Make sure your service handles that correctly with a 400, 404 or 405 and not a 500.

Reply:
No problem-
However, this is something that falls into the "default" error, right? Namely there shouldn't be a specific response defined here in the swagger for it?